<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="2" failed="2" skipped="1">
  <reporter-output>
    <line>
      <![CDATA[i am the main1]]>
    </line>
    <line>
      <![CDATA[i am the main2]]>
    </line>
  </reporter-output>
  <suite started-at="2024-10-22T13:18:32 IST" name="Suite" finished-at="2024-10-22T13:24:03 IST" duration-ms="331154">
    <groups>
    </groups>
    <test started-at="2024-10-22T13:18:32 IST" name="Test" finished-at="2024-10-22T13:24:03 IST" duration-ms="331154">
      <class name="ExtentReportBasic.Testing5">
        <test-method is-config="true" signature="preCondition()[pri:0, instance:ExtentReportBasic.Testing5@4efac082]" started-at="2024-10-22T13:18:32 IST" name="preCondition" finished-at="2024-10-22T13:22:25 IST" duration-ms="232756" status="FAIL">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[java.util.concurrent.TimeoutException
Build info: version: '4.18.1', revision: 'b1d3319b48'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.3'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [8ca3d774fc1407d65038aa203f1a8063, maximizeCurrentWindow {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 129.0.6668.101, chrome: {chromedriverVersion: 129.0.6668.100 (cf58cba358d..., userDataDir: C:\Users\Dell\AppData\Local...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:57035}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:57035/devtoo..., se:cdpVersion: 129.0.6668.101, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 8ca3d774fc1407d65038aa203f1a8063]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: java.util.concurrent.TimeoutException
Build info: version: '4.18.1', revision: 'b1d3319b48'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.3'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [8ca3d774fc1407d65038aa203f1a8063, maximizeCurrentWindow {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 129.0.6668.101, chrome: {chromedriverVersion: 129.0.6668.100 (cf58cba358d..., userDataDir: C:\Users\Dell\AppData\Local...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:57035}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:57035/devtoo..., se:cdpVersion: 129.0.6668.101, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 8ca3d774fc1407d65038aa203f1a8063
at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:403)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute(JdkHttpClient.java:359)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:187)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:519)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:592)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:596)
at org.openqa.selenium.remote.RemoteWebDriver$RemoteWebDriverOptions$RemoteWindow.maximize(RemoteWebDriver.java:1035)
at com.crm.Dwsbaseclass.DwsBase.preCondition(DwsBase.java:22)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: java.util.concurrent.TimeoutException
at java.base/java.util.concurrent.CompletableFuture.timedGet(CompletableFuture.java:1960)
at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2095)
at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:386)
... 35 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preCondition -->
        <test-method is-config="true" signature="login()[pri:0, instance:ExtentReportBasic.Testing5@4efac082]" started-at="2024-10-22T13:22:25 IST" name="login" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1729583545917" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="main5()[pri:0, instance:ExtentReportBasic.Testing5@4efac082]" started-at="2024-10-22T13:22:25 IST" name="main5" finished-at="2024-10-22T13:22:25 IST" duration-ms="0" status="SKIP">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[java.util.concurrent.TimeoutException
Build info: version: '4.18.1', revision: 'b1d3319b48'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.3'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [8ca3d774fc1407d65038aa203f1a8063, maximizeCurrentWindow {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 129.0.6668.101, chrome: {chromedriverVersion: 129.0.6668.100 (cf58cba358d..., userDataDir: C:\Users\Dell\AppData\Local...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:57035}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:57035/devtoo..., se:cdpVersion: 129.0.6668.101, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 8ca3d774fc1407d65038aa203f1a8063]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: java.util.concurrent.TimeoutException
Build info: version: '4.18.1', revision: 'b1d3319b48'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.3'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [8ca3d774fc1407d65038aa203f1a8063, maximizeCurrentWindow {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 129.0.6668.101, chrome: {chromedriverVersion: 129.0.6668.100 (cf58cba358d..., userDataDir: C:\Users\Dell\AppData\Local...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:57035}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:57035/devtoo..., se:cdpVersion: 129.0.6668.101, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 8ca3d774fc1407d65038aa203f1a8063
at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:403)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute(JdkHttpClient.java:359)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:187)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:519)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:592)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:596)
at org.openqa.selenium.remote.RemoteWebDriver$RemoteWebDriverOptions$RemoteWindow.maximize(RemoteWebDriver.java:1035)
at com.crm.Dwsbaseclass.DwsBase.preCondition(DwsBase.java:22)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: java.util.concurrent.TimeoutException
at java.base/java.util.concurrent.CompletableFuture.timedGet(CompletableFuture.java:1960)
at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2095)
at org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:386)
... 35 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- main5 -->
        <test-method is-config="true" signature="logout()[pri:0, instance:ExtentReportBasic.Testing5@4efac082]" started-at="2024-10-22T13:22:26 IST" name="logout" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1729583546403" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="postCondition()[pri:0, instance:ExtentReportBasic.Testing5@4efac082]" started-at="2024-10-22T13:22:26 IST" name="postCondition" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1729583546407" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postCondition -->
      </class> <!-- ExtentReportBasic.Testing5 -->
      <class name="ExtentReportBasic.Testing4">
        <test-method is-config="true" signature="preCondition()[pri:0, instance:ExtentReportBasic.Testing4@53aac487]" started-at="2024-10-22T13:22:26 IST" name="preCondition" finished-at="2024-10-22T13:23:02 IST" duration-ms="36092" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preCondition -->
        <test-method is-config="true" signature="login()[pri:0, instance:ExtentReportBasic.Testing4@53aac487]" started-at="2024-10-22T13:23:02 IST" name="login" finished-at="2024-10-22T13:23:09 IST" duration-ms="6841" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="main4()[pri:0, instance:ExtentReportBasic.Testing4@53aac487]" started-at="2024-10-22T13:23:09 IST" name="main4" finished-at="2024-10-22T13:23:09 IST" duration-ms="155" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [big] but found [bug]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [big] but found [bug]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at ExtentReportBasic.Testing4.main4(Testing4.java:12)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- main4 -->
        <test-method is-config="true" signature="logout()[pri:0, instance:ExtentReportBasic.Testing4@53aac487]" started-at="2024-10-22T13:23:09 IST" name="logout" finished-at="2024-10-22T13:23:11 IST" duration-ms="1264" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="postCondition()[pri:0, instance:ExtentReportBasic.Testing4@53aac487]" started-at="2024-10-22T13:23:11 IST" name="postCondition" finished-at="2024-10-22T13:23:13 IST" duration-ms="2337" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postCondition -->
      </class> <!-- ExtentReportBasic.Testing4 -->
      <class name="ExtentReportBasic.Testing3">
        <test-method is-config="true" signature="preCondition()[pri:0, instance:ExtentReportBasic.Testing3@273e7444]" started-at="2024-10-22T13:23:30 IST" name="preCondition" finished-at="2024-10-22T13:23:41 IST" duration-ms="11105" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preCondition -->
        <test-method is-config="true" signature="login()[pri:0, instance:ExtentReportBasic.Testing3@273e7444]" started-at="2024-10-22T13:23:41 IST" name="login" finished-at="2024-10-22T13:23:44 IST" duration-ms="3299" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="main3()[pri:0, instance:ExtentReportBasic.Testing3@273e7444]" started-at="2024-10-22T13:23:44 IST" name="main3" finished-at="2024-10-22T13:23:44 IST" duration-ms="6" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [than] but found [thon]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [than] but found [thon]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at ExtentReportBasic.Testing3.main3(Testing3.java:12)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- main3 -->
        <test-method is-config="true" signature="logout()[pri:0, instance:ExtentReportBasic.Testing3@273e7444]" started-at="2024-10-22T13:23:44 IST" name="logout" finished-at="2024-10-22T13:23:46 IST" duration-ms="1435" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="postCondition()[pri:0, instance:ExtentReportBasic.Testing3@273e7444]" started-at="2024-10-22T13:23:46 IST" name="postCondition" finished-at="2024-10-22T13:23:47 IST" duration-ms="1102" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postCondition -->
      </class> <!-- ExtentReportBasic.Testing3 -->
      <class name="ExtentReportBasic.Testing2">
        <test-method is-config="true" signature="preCondition()[pri:0, instance:ExtentReportBasic.Testing2@7db12bb6]" started-at="2024-10-22T13:23:47 IST" name="preCondition" finished-at="2024-10-22T13:23:58 IST" duration-ms="10872" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preCondition -->
        <test-method is-config="true" signature="login()[pri:0, instance:ExtentReportBasic.Testing2@7db12bb6]" started-at="2024-10-22T13:23:58 IST" name="login" finished-at="2024-10-22T13:24:00 IST" duration-ms="2531" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="main2()[pri:0, instance:ExtentReportBasic.Testing2@7db12bb6]" started-at="2024-10-22T13:24:00 IST" name="main2" finished-at="2024-10-22T13:24:00 IST" duration-ms="4" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[i am the main2]]>
            </line>
          </reporter-output>
        </test-method> <!-- main2 -->
        <test-method is-config="true" signature="logout()[pri:0, instance:ExtentReportBasic.Testing2@7db12bb6]" started-at="2024-10-22T13:24:00 IST" name="logout" finished-at="2024-10-22T13:24:02 IST" duration-ms="1666" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="postCondition()[pri:0, instance:ExtentReportBasic.Testing2@7db12bb6]" started-at="2024-10-22T13:24:02 IST" name="postCondition" finished-at="2024-10-22T13:24:03 IST" duration-ms="1226" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postCondition -->
      </class> <!-- ExtentReportBasic.Testing2 -->
      <class name="ExtentReportBasic.Testing1">
        <test-method is-config="true" signature="preCondition()[pri:0, instance:ExtentReportBasic.Testing1@52b1beb6]" started-at="2024-10-22T13:23:13 IST" name="preCondition" finished-at="2024-10-22T13:23:24 IST" duration-ms="11130" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preCondition -->
        <test-method is-config="true" signature="login()[pri:0, instance:ExtentReportBasic.Testing1@52b1beb6]" started-at="2024-10-22T13:23:24 IST" name="login" finished-at="2024-10-22T13:23:27 IST" duration-ms="2984" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="main1()[pri:0, instance:ExtentReportBasic.Testing1@52b1beb6]" started-at="2024-10-22T13:23:27 IST" name="main1" finished-at="2024-10-22T13:23:27 IST" duration-ms="10" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[i am the main1]]>
            </line>
          </reporter-output>
        </test-method> <!-- main1 -->
        <test-method is-config="true" signature="logout()[pri:0, instance:ExtentReportBasic.Testing1@52b1beb6]" started-at="2024-10-22T13:23:27 IST" name="logout" finished-at="2024-10-22T13:23:29 IST" duration-ms="1765" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="postCondition()[pri:0, instance:ExtentReportBasic.Testing1@52b1beb6]" started-at="2024-10-22T13:23:29 IST" name="postCondition" finished-at="2024-10-22T13:23:30 IST" duration-ms="671" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postCondition -->
      </class> <!-- ExtentReportBasic.Testing1 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
